src/store/trails/mutations.js

```diff
@@ -53,15 +52,17 @@ export const setBallot = (state, ballot) => {
 
 export const setUserTreasuries = (state, treasuries) => {
   state.userTreasuries = treasuries;
+  console.log("Trails.mutations.setUserTreasuries() ", treasuries.rows);
 };
 
 export const setBallotVotes = (state, voters) => {
   state.ballotVoters = voters;
 };
 
 export const resetTreasuries = (state) => {
-  state.treasuries.list.data = [];
-  state.treasuries.list.loaded = false;
+  // same as resetBallos
+  // state.treasuries.list.data = [];
+  // state.treasuries.list.loaded = false;
 };
 
```

Comment:
Same here, if we aren't using the method, we can just remove it.

Reply:
Please add to new issue as it does not seem to be a blocker for relaunch.